### PR TITLE
Master point of sale remove workflow dve

### PR DESCRIPTION
**Pull request by Icallhimtest**
odoo task : https://www.odoo.com/web#id=27207&view_type=form&model=project.task&action=333&active_id=248&menu_id=5195

### CODE REVIEW

File: addons/point_of_sale/models/pos_session.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def _confirm_orders(self):
             for order in session.order_ids.filtered(lambda o: o.state != 'done'):
                 if order.state not in ('paid', 'invoiced'):
                     raise UserError(_("You cannot confirm all orders of this session, because they have not the 'paid' status"))
-                order.signal_workflow('done')
+                order.action_pos_order_done()
```
Comment (celm1990) [inline addons/point_of_sale/models/pos_session.py]: Please check this issue #15170

Comment (Icallhimtest) [inline addons/point_of_sale/models/pos_session.py]: Thanks for commenting here @celm1990

Indeed, it looks like orders in state 'invoiced' shouldn't get set to 'done' but remain 'invoiced' when closing the session. Seems a bit odd to me, but I'll fix it.

Comment (wtaferner) [inline addons/point_of_sale/models/pos_session.py]: @Icallhimtest 
Are you now the new responsible for PoS?
As already said, https://github.com/odoo/odoo/pull/15178 does it a better way...it does not make sense to test against invoiced orders if they are paid...btw. this is another regression in v10 not yet fixed (see the PR text)

Comment (Icallhimtest) [inline addons/point_of_sale/models/pos_session.py]: (I'm not the new reponsible for PoS, just fixed the bug I introduced during the workflow removal.)

**Pull request closed without merging**